packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -673,6 +678,102 @@ describe("Integration tests", () => {
         ).rejects.toThrowError(mailNotFound(mailIdNotInTenant));
       });
     });
+    describe("addTenantMail", async () => {
+      const mailSeed: MailSeed = {
+        kind: "CONTACT_EMAIL",
+        address: "testMail@test.it",
+        description: "mail description",
+      };
+
+      it("Should correctly add the mail", async () => {
+        vi.useFakeTimers();
+        vi.setSystemTime(new Date());
+        await addOneTenant(mockTenant, postgresDB, tenants);
+        await tenantService.addTenantMail({
+          tenantId: mockTenant.id,
+          mailSeed,
+          organizationId: mockTenant.id,
+          correlationId: generateId(),
+        });
+        const writtenEvent = await readLastTenantEvent(
+          mockTenant.id,
+          postgresDB
+        );
+        if (!writtenEvent) {
+          fail("Creation fails: tenant not found in event-store");
+        }
+
+        expect(writtenEvent).toMatchObject({
+          stream_id: mockTenant.id,
+          version: "1",
+          type: "TenantMailAdded",
+        });
+
+        const writtenPayload: TenantMailAddedV2 | undefined = protobufDecoder(
+          TenantMailAddedV2
+        ).parse(writtenEvent.data);
+
+        const updatedTenant: Tenant = {
+          ...mockTenant,
+          mails: [
+            {
+              id: writtenPayload.mailId,
+              createdAt: new Date(),
+              kind: "CONTACT_EMAIL",
+              address: "testMail@test.it",
+              description: "mail description",
+            },
+          ],
+          updatedAt: new Date(),
+        };
+        expect(writtenPayload.tenant).toEqual(toTenantV2(updatedTenant));
+        vi.useRealTimers();
+      });
+      it("Should throw tenantNotFound if the tenant doesn't exists", async () => {
+        expect(
+          tenantService.addTenantMail({
+            tenantId: mockTenant.id,
+            mailSeed,
+            organizationId: mockTenant.id,
+            correlationId: generateId(),
+          })
+        ).rejects.toThrowError(tenantNotFound(mockTenant.id));
+      });
+      it("Should throw operationForbidden when tenantId is not the organizationId", async () => {
+        await addOneTenant(mockTenant, postgresDB, tenants);
+        expect(
+          tenantService.addTenantMail({
+            tenantId: mockTenant.id,
+            mailSeed,
+            organizationId: generateId(),
+            correlationId: generateId(),
+          })
+        ).rejects.toThrowError(operationForbidden);
+      });
+      it("Should throw mailAlreadyExists if address already exists in the tenant mail", async () => {
+        const tenant: Tenant = {
+          ...mockTenant,
+          mails: [
+            {
+              id: generateId(),
+              createdAt: new Date(),
+              kind: "CONTACT_EMAIL",
+              address: "testMail@test.it",
```

Comment:
```suggestion
              ...mailSeed
```
You can just add what's inside `mailSeed` in order to be sure that it's the same mail that we are adding